migrations/sql/schema/schema_6.0.0.py

```diff
@@ -0,0 +1,66 @@
+"""Migration script for schema 6.0.0.
+
+Revision ID: schema_6.0.0
+Revises: schema_6.0.0
+Create Date: 2025-02-18 12:06:42.740977
+
+"""
+
+import logging
+
+import sqlalchemy
+from lsst.dax.apdb_migrate.sql.context import Context
+
+# revision identifiers, used by Alembic.
+revision = "schema_6.0.0"
+down_revision = "schema_5.0.0"
+branch_labels = None
+depends_on = None
+
+_LOG = logging.getLogger(__name__)
+
+
+def upgrade() -> None:
+    """Upgrade 'schema' tree from 5.0.0 to 6.0.0 (ticket DM-45621).
+    Summary of changes:
+      - Add empty pixelFlags_nodata and pixelFlags_nodataCenter columns.
+    """
+    ctx = Context()
+
+    # Alter table schema.
+    _LOG.info("Adding columns to DiaSource table.")
+    try:
+        summary = ctx.get_table("DiaSource")
+        with ctx.batch_alter_table("DiaSource", copy_from=summary) as batch_op:
+            # add empty column.
+            batch_op.add_column(sqlalchemy.Column("is_negative", sqlalchemy.types.Boolean, nullable=True))
```

Comment:
@parejkoj, I think this should have been `pixelFlags_nodataCenter` instead of `is_negative`?

Reply:
Ah, shoot, yes you are absolutely correct! I didn't know if there was a way to test this. Please push a hotfix for that, and I'm sorry.